test-lib-openshift.sh

```diff
@@ -825,7 +825,13 @@ function ct_os_test_template_app_func() {
       for i_t in ${other_images} ; do
         echo "${i_t}"
         IFS='|' read -ra image_tag_a <<< "${i_t}"
-        docker pull "${image_tag_a[0]}"
+        if [[ "$(docker images -q "$image_name" 2>/dev/null)" != "" ]]; then
```

Comment:
In case the image is already pulled, then this condition is wrong.

It should be `if [[ "$(docker images -q "$image_name" 2>/dev/null)" == "" ]]; then`